src/cozy/wrappers/sharing/sharing.component.ts

```diff
@@ -148,7 +148,18 @@ export class SharingComponent extends AngularWrapperComponent {
     }
 
     protected async rejectUser(user: User) {
-        // console.log('rejectUser', user)
+        try {
+            const client = this.clientService.GetClient();
+
+            await client.stackClient.fetchJSON(

```

Comment:
Une raison particulière de ne pas utiliser `client.destroy` ?

Reply:
Oui, lorsque j'essaie d'utiliser destroy j'ai l'erreur que je t'ai montré l'autre jour : `ERROR Error: Uncaught (in promise): FetchError: {"error":"code=403, message=reserved doctype com.bitwarden.contacts unwritable"} FetchError: {"error":"code=403, message=reserved doctype com.bitwarden.contacts unwritable"}`

La route habituelle de gestion des doctypes ne semble pas branchée et il faut donc passer par `/bitwarden/contacts/:ID`.

Reply:
Ok, je manque de connaissance sur `cozy-client`, mais c'est bon à savoir..